clients/java/src/main/java/io/camunda/client/api/search/response/UserTask.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.client.api.search.response;
+
+import io.camunda.client.api.response.UserTaskEvent;
+
+public interface UserTask extends UserTaskEvent {
+
+  @Override
+  String getCreationDate();
+
+  @Override
+  String getCompletionDate();
```

Comment:
❌ If the interface here extends the `UserTaskEvent`, it should only add new methods, not override existing ones. Otherwise, the interface here is rather useless. If you want to keep the separation between the two interfaces, remove the `String getCompletionDate();` and `String getState();` from the `UserTaskEvent` and add them here in this interface.